### PR TITLE
Add DocsTub

### PR DESCRIPTION
**Pull request by vishal-sancheti**
This is a new submission to built with react.

### CODE REVIEW

**Comment by vishal-sancheti** [comment]
Any Updates for the Pull ?

File: CNAME [hunk 1]
```diff
@@ -1 +0,0 @@
-builtwithreact.io
```
Comment (jpadilla) [inline CNAME:1]: This must stay.

**Comment by vishal-sancheti** [comment]
Please check now !

**Comment by vishal-sancheti** [comment]
Thanks for merging :)

**Pull request merged**